.github/workflows/test.yaml

```diff
@@ -0,0 +1,39 @@
+name: Test
+on: [push]
+jobs:
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        minor_version: [8, 9, 10, 11, 12]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
+        with:
+          python-version: 3.${{ matrix.minor_version }}
+      - name: update pip
+        run: |
+          pip install -U wheel
+          pip install -U setuptools
+          python -m pip install -U pip
+      - run: pip install tox
+      - run: tox -e py3${{ matrix.minor_version }}
```

Comment:
there aren't any tests in this repo right now :D

Reply:
yes, there should and will be, once we submit the revision by Dec 10th.